TYPO3.Flow/Classes/TYPO3/Flow/Http/Request.php

```diff
@@ -71,7 +91,24 @@ class Request extends AbstractMessage
      */
     public function __construct(array $get, array $post, array $files, array $server)
     {
+        if (self::$trustedProxiesSettings === null) {
+            self::$trustedProxiesSettings = [
+                'proxies' => '*',
+                'headers' => [
+                    self::HEADER_CLIENT_IP => 'X-Forwarded-For',
+                    self::HEADER_HOST => 'X-Forwarded-Host',
+                    self::HEADER_PORT => 'X-Forwarded-Port',
+                    self::HEADER_PROTOCOL => 'X-Forwarded-Proto'
+                ]
+            ];
+            if (Bootstrap::$staticObjectManager !== null) {
+                $configurationManager = Bootstrap::$staticObjectManager->get(ConfigurationManager::class);
+                self::$trustedProxiesSettings = $configurationManager->getConfiguration(ConfigurationManager::CONFIGURATION_TYPE_SETTINGS, 'TYPO3.Flow.http.trustedProxies');
```

Comment:
> Also thought about a merge, but that would possibly create unwanted merge results.

Mh.. I think it can also be confusing to suddenly replace the settings during runtime.. Again, if its about tests lets try to get the tests deal with this (be it by manually setting `Bootstrap::$staticObjectManager` where it would break otherwise  (or extract that part out to a method like `getTrustedProxiesSettings()` and mock that in tests)